### DISCUSSION TITLE
How do I force a long list of authors to wrap?

### DISCUSSION THREAD

**Discussion by robjhyndman** [quarto-dev/quarto-cli#9531] [Q&A]
### Description

On my website, I have some articles with a long list of authors. Here is an example: http://robjhyndman.com/publications/covid19b/
In case I caused this with my own css styling, I tried making a freshly minted quarto website with no tweaking, and the same problem occurs. So it's not just me.

I guess some changes are needed to the template partials or css, but I'm not sure what is required.

**Comment by mcanouil** [comment]
> In case I caused this with my own css styling, I tried making a freshly minted quarto website with no tweaking, and the same problem occurs. So it's not just me.

Could you share that?

**Reply by robjhyndman** [reply]
https://github.com/robjhyndman/test-many-authors

**Reply by cscheid** [reply]
A temporary workaround (that doesn't look _that_ much better) is to switch to `authors` instead, and use an array of names:

````
---
title: Test
date: "2023-05-19"
authors:
  - Robert Moss
  - David J Price
  - Nick Golding
  - Peter Dawson
  - Jodie McVernon
  - Rob J Hyndman
  - Freya M Shearer
  - James M McCaw
  - Robert Moss
  - David J Price
  - Nick Golding
  - Peter Dawson
  - Jodie McVernon
  - Rob J Hyndman
  - Freya M Shearer
  - James M McCaw
````

That ends up looking too tall, so I don't love that either. I think this is a bug we need to fix.

**Reply by cscheid** [reply]
That's fixed on `main` now. Thanks for reporting!

**Comment by mcanouil** [comment]
- https://github.com/quarto-dev/quarto-cli/issues/9539
